README.md

```diff
@@ -23,4 +25,20 @@ Run with `go test -benchmem -benchtime 5s -bench .`
 | ProtoHTTP1_1K  |  20000 |  383342 ns/op |  5.34 MB/s |  55044 B/op | 416 allocs/op |
 | ProtoHTTP1_64K |   3000 | 1932455 ns/op | 67.83 MB/s | 827150 B/op | 377 allocs/op |
 
+## go version devel +59129c6 Fri Sep 25 22:25:52 2015 +0000 darwin/amd64
+
+These are faster than go1.5.1 thanks to cloudflare's amd64 assembly implementation of
+AES-GCM: https://github.com/golang/go/commit/efeeee3.
+
+| Bench          | Iters  | Speed         | Throughput  | Allocations | Allocations    |
```

Comment:
an old version of this had unencrypted numbers; HTTP2 is TLS-only, so I had to even the playing field but changing everything to TLS. Old numbers: https://github.com/cockroachdb/rpc-bench/tree/d2235332f4f593f6c98653f1574855d38bb90a8f